compiler/rustc_middle/src/query/mod.rs

```diff
@@ -1136,6 +1136,7 @@ rustc_queries! {
         key: ty::ParamEnvAnd<'tcx, (ty::PolyFnSig<'tcx>, &'tcx ty::List<Ty<'tcx>>)>
     ) -> Result<&'tcx abi::call::FnAbi<'tcx, Ty<'tcx>>, ty::layout::FnAbiError<'tcx>> {
         desc { "computing call ABI of `{}` function pointers", key.value.0 }
+        anon
```

Comment:
Is this entirely for performance purposes? It sounds to me like it should be undesirable to enable this.

Reply:
Hmm, did I keep a HACK comment around? This was for testing perf and can be removed.

Reply:
There's a HACK in the commit message.